packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -10,7 +10,9 @@
   },
   "dependencies": {
     "@salesforce/core": "2.28.0",
-    "shelljs": "0.8.5"
+    "shelljs": "0.8.5",
+    "jsforce": "1.10.0",
```

Comment:
same comment...should all jsforce be 1.11.0?

Reply:
Done